### PR TITLE
add fusor.rss() and SwingAcceleration<>

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: common/fuse.h [hunk 1]
```diff
@@ -290,6 +290,13 @@ class Fusor : public Looper {
   Vec3 speed() { return speed_; }  // m/s
 #endif
 
+  float rss() {
+    Vec3 gyro_slope = gyro_extrapolator_.slope() * 1000000;
```
Comment (profezzorn) [inline common/fuse.h:294]: This should just call gyro_slope()

File: common/fuse.h [hunk 2]
```diff
@@ -290,6 +290,13 @@ class Fusor : public Looper {
   Vec3 speed() { return speed_; }  // m/s
 #endif
 
+  float rss() {
```
Comment (profezzorn) [inline common/fuse.h:293]: Needs a comment to explain what it does.
(Or a MUCH better name....)

File: common/fuse.h [hunk 3]
```diff
@@ -290,6 +290,13 @@ class Fusor : public Looper {
...
+    Vec3 gyro_slope = gyro_extrapolator_.slope() * 1000000;
+    // Radians per second per second
+    float rss = sqrtf(gyro_slope.z * gyro_slope.z + gyro_slope.y * gyro_slope.y) * (M_PI / 180);
```
Comment (profezzorn) [inline common/fuse.h:296]: variable is not needed, just do: return sqrtf(....) * (...);

File: functions/swing_speed.h [hunk 4]
```diff
@@ -26,4 +26,27 @@ using SwingSpeedX = SingleValueAdapter<SwingSpeedSVF<MAX>>;
...
+// Usage: SwingAcceleration<MAX>
+// Returns 0-32768 based on swing acceleration
+// MAX defaults to 150
+// returned value: INTEGER
+template<class MAX = Int<130>>
+class SwingAccelerationSVF {
+ public:
+  void run(BladeBase* blade) { max_.run(blade); }
+ int calculate(BladeBase* blade) {
+   max_.run(blade);
```
Comment (profezzorn) [inline functions/swing_speed.h:38]: No need to call max_.run() again. (It's already done in SwingAccelerationSVF::run() above)

File: common/fuse.h (hunk 1 above)
Comment (fdarosa2663) [reply common/fuse.h]: done

File: common/fuse.h (hunk 2 above)
Comment (fdarosa2663) [reply common/fuse.h]: renamed to swing_accel() and added comment

File: common/fuse.h (hunk 3 above)
Comment (fdarosa2663) [reply common/fuse.h]: updated

File: functions/swing_speed.h (hunk 4 above)
Comment (fdarosa2663) [reply functions/swing_speed.h]: remove duplication in SwingSpeed as well

**Pull request merged**